t/v28_features.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/python
+# -_- encoding: utf8 -_-
+
+from http_utils import *
+
+# =============
+#
+preset_method_location = BASE_URL + '/skipping_test'
+body = [
+  {
+    "skipping_map_at_the_beginning": {
+      "ttl": 123
+    },
+    "id": 1,
+    "method": "skipping_test",
+    "params": []
+  },
+  {
+    "id": 1,
+    "skipping_map_in_the_middle": {
+      "ttl": 123
+    },
+    "method": "skipping_test",
+    "params": []
+  },
+  {
+    "id": 1,
+    "method": "skipping_test",
+    "params": [],
+    "skipping_map_in_the_end": {
+      "ttl": 123
+    }
+  },
+  {
+    "skipping_array_at_the_beginning": [
+      "ttl",
+      123
+    ],
+    "id": 1,
+    "method": "skipping_test",
+    "params": [],
+    "meta": [
+      "ttl",
+      123
+    ]
+  },
+  {
+    "id": 1,
+    "method": "skipping_test",
+    "skipping_array_in_the_middle": [
+      "ttl",
+      123
+    ],
+    "params": []
+  },
+  {
+    "id": 1,
+    "method": "skipping_test",
+    "params": [],
+    "skipping_array_in_the_end": [
+      "ttl",
+      123
+    ]
+  },
+  {
+    "skipping_map_with_nesting": {
+      "ttl": {
+        "ttl": 123,
+        "array":[]
+        }
+    },
+    "id": 1,
+    "method": "skipping_test",
+    "params": []
+  },
+  {
+    "skipping_array_with_nesting": [
+      "ttl",
+      [
+        "ttl",
+        {
+          "map":
+          123
+        }
+      ]
+    ],
+    "id": 1,
+    "method": "skipping_test",
+    "params": []
+  }
+]
+result = post_success_pure(preset_method_location, body, {})
+for a in result:
```

Comment:
Please try not to use a single letter name (except for the widespread ones like i, j, etc.)

Reply:
Done.